index.html

```diff
@@ -54,5 +57,174 @@ <h1>I'm Fernando Glad to see <br> you!</h1>
+				<img src="img/img1.jpg" alt="" />
+				<h1>Tonic</h1>
+				<div class="card-details">
+					<h3>canopy</h3>
+					<i class="fas fa-circle"></i>
+					<h4>back end dev</h4>
+					<i class="fas fa-circle"></i>
+					<h4>2015</h4>
+				</div>
+				<p class="card-description">
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<div class="card-skills">
+					<ul>
+						<li><a href="#">html</a></li>
+						<li><a href="#">css</a></li>
+						<li><a href="#">javaScript</a></li>
+					</ul>
+				</div>
+				<button class="btn">see project</button>
+			</div>
+
+			<div class="card">
+				<img src="img/img2.jpg" alt="" />
+				<h1>Multi-Post Stories</h1>
+				<div class="card-details">
+					<h3>canopy</h3>
+					<i class="fas fa-circle"></i>
+					<h4>back end dev</h4>
+					<i class="fas fa-circle"></i>
+					<h4>2015</h4>
+				</div>
+				<p class="card-description">
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<div class="card-skills">
+					<ul>
+						<li><a href="#">html</a></li>
+						<li><a href="#">css</a></li>
+						<li><a href="#">javaScript</a></li>
+					</ul>
+				</div>
+				<button class="btn">see project</button>
+			</div>
+
+			<div class="card">
+				<img src="img/img3.jpg" alt="" />
+				<h1>Tonic</h1>
+				<div class="card-details">
+					<h3>canopy</h3>
+					<i class="fas fa-circle"></i>
+					<h4>back end dev</h4>
+					<i class="fas fa-circle"></i>
+					<h4>2015</h4>
+				</div>
+				<p class="card-description">
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<div class="card-skills">
+					<ul>
+						<li><a href="#">html</a></li>
+						<li><a href="#">css</a></li>
+						<li><a href="#">javaScript</a></li>
+					</ul>
+				</div>
+				<button class="btn">see project</button>
+			</div>
+
+			<div class="card">
+				<img src="img/img4.jpg" alt="" />
+				<h1>Multi-Post Stories</h1>
+				<div class="card-details">
+					<h3>canopy</h3>
+					<i class="fas fa-circle"></i>
+					<h4>back end dev</h4>
+					<i class="fas fa-circle"></i>
+					<h4>2015</h4>
+				</div>
+				<p class="card-description">
+					A daily selection of privately personalized reads; no accounts or
+					sign-ups required.
+				</p>
+				<div class="card-skills">
+					<ul>
+						<li><a href="#">html</a></li>
+						<li><a href="#">css</a></li>
+						<li><a href="#">javaScript</a></li>
+					</ul>
+				</div>
+				<button class="btn">see project</button>
+			</div>
+		</section>
+
+		<section class="about">
+			<h1>
+				About <br />
+				Myself
+			</h1>
+			<p>
+				Hello I’m a software developer! I can help you build a product , feature
+				or website Look through some of my work and experience! If you like what
+				you see and have a project you need coded, don’t hestiate to contact me.
+			</p>
+			<a href="#" class="connect">let's connect</a>
```

Comment:
- [ ] Please consider replacing the `a` tag here with the `h2` tag as this text is not a link.

```suggestion
			</p>
			<h2 class="connect">let's connect</h2>
```